lib/apipie/response_description.rb

```diff
@@ -0,0 +1,113 @@
+module Apipie
+
+  class ResponseObject
```

Comment:
Either nest it under `ResponseDescription` or put it into separate file, so that the file naming follows the class names